randovania/games/prime3/logic_database/Pirate Homeworld - Command.txt

```diff
@@ -153,8 +153,8 @@ Extra - asset_id: 18221971457754325372
   * Layers: default
   > Door to Courtyard Passage
       All of the following:
-          # https://www.youtube.com/watch?v=N6gHBtfWj2w
-          Bomb/Spring Space Jump (Advanced) and Use Screw Attack (Space Jump)
+          # https://youtu.be/N6gHBtfWj2w
+          Bomb/Spring Space Jump (Advanced) and Standable Terrain (Intermediate) and Use Screw Attack (Space Jump)
```

Comment:
Does this also need 'Single-room out of bounds'?

Reply:
I don't think so. It's just that the standable is higher than you would normally. You aren't dealing with any backfaced collision or aether.